### ISSUE TITLE
UserWarning: Gradients will be none

### ISSUE THREAD

**Issue by rektobot** [TheLastBen/fast-stable-diffusion#1623]
never got one of these before  but start-up cell has been spitting out a lot of python & torch issues the past two days:

UserWarning: none of the inputs have requires_grad=true
Gradients will be none

**Comment by TheLastBen** [comment]
this must be the dreambooth extension, right ?

**Comment by rektobot** [comment]
no, auto1111

**Comment by TheLastBen** [comment]
Are you using the latest notebook ?

**Comment by rektobot** [comment]
yeah im running the latest notebook, i used the main link yesterday and it updated with the controlnet cell but i already had controlnet installed so i dont use that cell. everything runs atm, i just get those tensor errors and nan-check errors randomly. i ran the colab the other day without using gdrive and had no issues at all and its much faster it seems cause streaming files sucks and gdrive is absolute dogshit. all the errors im getting for python and tensor are all the local installed stuff on the colab end.

**Comment by TheLastBen** [comment]
You might want to do some cleaning in your SD folder and update the extensions
